api/user.go

```diff
@@ -1510,6 +1510,8 @@ func UpdateActive(c *Context, user *model.User, active bool) *model.User {
 			RevokeAllSession(c, user.Id)
 		}
 
+		<-Srv.Store.Channel().ExtraUpdateByUser(user.Id, model.GetMillis())
+
```

Comment:
I did avoid the `return` cause that action doesn't prevent the user for being updated.